trame_vtk/modules/paraview/protocols/local_rendering.py

```diff
@@ -0,0 +1,108 @@
+from paraview import simple
+from wslink import register as exportRpc
+
+from trame_vtk.modules.vtk.protocols.serializers import (
```

Comment:
serializers are independent of protocols... They should be siblings.

Reply:
Done.